apps/member-profile/app/routes/api.company-review.$id.downvote.ts

```diff
@@ -0,0 +1,15 @@
+import { type ActionFunctionArgs, json } from '@remix-run/node';
```

Comment:
There was a mistake on my part. I called the files and the function “downvote a company review” but what I really meant was “undo a company upvote”. So those files/function aren’t really downvoting anything. They are just making sure to undo an upvote when the user clicks on it. So you can upvote and undo your upvote in case you clicked the upvote by accident. Does that make sense?

Should I move the functionality to the same file as upvote company or should I just rename the files?